src/legacy/core_plugins/kibana/server/lib/management/saved_objects/relationships.js

```diff
@@ -1,64 +0,0 @@
-/*
- * Licensed to Elasticsearch B.V. under one or more contributor
- * license agreements. See the NOTICE file distributed with
- * this work for additional information regarding copyright
- * ownership. Elasticsearch B.V. licenses this file to you under
- * the Apache License, Version 2.0 (the "License"); you may
- * not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *    http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing,
- * software distributed under the License is distributed on an
- * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
- * KIND, either express or implied.  See the License for the
- * specific language governing permissions and limitations
- * under the License.
- */
-
-import { pick } from 'lodash';
-import { injectMetaAttributes } from './inject_meta_attributes';
-
-export async function findRelationships(type, id, options = {}) {
-  const { size, savedObjectsClient, savedObjectTypes, savedObjectsManagement } = options;
-
-  const { references = [] } = await savedObjectsClient.get(type, id);
```

Comment:
Not directly related to this issue, but some files in `src/legacy/core_plugins/kibana/server/lib/management` were not deleted when migrating the SavedObject management section. I removed them.